spec/acceptance/scl_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper_acceptance'
+
+if fact('osfamily') == 'RedHat'
+  describe 'Scenario: install scl-bind' do
+
+    it_behaves_like 'an idempotent resource' do
+      let(:manifest) do
+        <<-EOS
+        yumrepo{'bind_scl':
+          descr    => 'copr repo for bind owned by isc'
+          baseurl  =>  "https://download.copr.fedorainfracloud.org/results/isc/bind/epel-$facts['os']['release']['major']-x86_64/"
+          gpgcheck =>  0
```

Comment:
I think you're missing commas here
```suggestion
          descr    => 'copr repo for bind owned by isc',
          baseurl  => "https://download.copr.fedorainfracloud.org/results/isc/bind/epel-$facts['os']['release']['major']-x86_64/",
          gpgcheck => 0,
```